src/main/webapp/index.html

```diff
@@ -3,9 +3,25 @@
   <head>
     <meta charset="UTF-8">
     <title>SPS summer 21 team 18</title>
+      <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBriew9kNL5KzMqM4LwqUhYx_5LisEI2go"></script>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+
+  <header>
+      <h1>Homepage</h1>
+  </header>
+
+  <article>
```

Comment:
great job, Kiana.
Let use space of 2 instead of tabs. as google style guide recommends.
https://google.github.io/styleguide/htmlcssguide.html#Indentation